### ISSUE TITLE
Few bugs on Teaching Kits

### ISSUE THREAD

**Issue by ZenanZha** [mozilla/thimble.mozilla.org#2033]
Some unimportant bugs on Teaching Kits:
1. The first two Teaching Kits on the homepage cannot be remixed.
![step1](https://cloud.githubusercontent.com/assets/22037167/25303449/36bacdc4-2721-11e7-9eb8-3ae8cd6ef19f.gif)

2. The remix button on the "Three Things I <3 Teaching Kit" is broken.
![step2](https://cloud.githubusercontent.com/assets/22037167/25303459/6e5431ee-2721-11e7-9274-9b9d2177b55c.gif)

3. The Teaching Kit under "Current Events Comic Strip Teaching Kit" has an unexpected tutorial.html.(which is from other Starting Kit)
![step3](https://cloud.githubusercontent.com/assets/22037167/25303502/3ec66bb2-2722-11e7-9d9c-53e2fe07ce6f.gif)

**Comment by ZenanZha** [comment]
@flukeout 
Any suggestion on this issue?

**Comment by flukeout** [comment]
Yeah, our teaching kits all use different kinds of templates. I think we have a Thimble-based template for the teaching kits, I'll dig it up and maybe we can slowly start converting the old style teaching kits into the new one.

**Comment by jgmac1106** [comment]
I believe these are the current kits:  curriculum kits for the unit:
https://thimbleprojects.org/chadsansing/192408/index.html and individual
lessons: https://d157rqmxrxj6ey.cloudfront.net/flukeout/17698/#step-1 though
looking at url from individual lesson it needs an update in url
